test/integration/test_ibm_integration.py

```diff
@@ -76,8 +76,8 @@ def test_compile_remote(self):
         quantum_circuit.h(qubit_reg[0])
         quantum_circuit.cx(qubit_reg[0], qubit_reg[1])
         quantum_circuit.measure(qubit_reg, clbit_reg)
-
-        circuits = transpile(quantum_circuit, backend=self.real_device_backend)
+        backend = self.dependencies.provider.get_backend("ibmq_lima")
+        circuits = transpile(quantum_circuit, backend=backend)
```

Comment:
if a custom backend is not set, `ibmq_armonk`, which is usually the least busy backend is used which will result in an error since armonk only has 2 qubits

Reply:
We should add min_num_qubits to @integration_test_setup_with_backend and then use that when getting least_busy backend, that way we can specify min num of qubits required for our test instead of hard coding a specific backend.

Reply:
Updated